### DISCUSSION TITLE
"Harvey balls" for to-do progress tracking

### DISCUSSION THREAD

**Discussion by night-tripper** [ransome1/sleek#497] [Ideas]
Firstly, thanks a lot to everyone involved in the development of this wonderful tool.

Although the simplicity of sleek is one of the reasons I like it so much, I was wondering if the incorporation of Harvey balls as an optional feature to track progress of tasks could be helpful:
![image](https://github.com/ransome1/sleek/assets/117457566/4cea87d4-55aa-418c-ab2d-17f41b01b406)

I would use them for indicating if a to-do has been started or progressed and would guess that 0% / 25% / 50% / 75% intervals would suffice.

They could be placed next to the tags or could be combined with the tick box. 

Curious to hear what you think!

**Comment by ransome1** [comment]
@night-tripper  thanks for proposing this. How would we reflect this in the todo syntax?
